main.py

```diff
@@ -19,7 +19,12 @@
 import MapBox
 import RocketData
 from RocketData import RocketData as RD
-import mplwidget  # DO NOT REMOVE pyinstller needs this
+
+import _thread
+import asyncio
+import mplwidget #DO NOT REMOVE pyinstller needs this
+
+import threading
```

Comment:
```suggestion
import mplwidget #DO NOT REMOVE pyinstller needs this
```
I don't think you need any of this anymore